katello/katello/katello-service

```diff
@@ -2,6 +2,14 @@
 
 require 'optparse'
 
+# check if production install
```

Comment:
This feels very hacky and fragile. If we're going to use it as a library, I'm strongly leaning to making it a properly installed library.